specification/programmableconnectivity/ProgrammableConnectivity.Management/Gateway.tsp

```diff
@@ -51,6 +51,312 @@ model GatewayProperties {
    */
   @visibility("read")
   provisioningState?: ProvisioningState;
+
+  /**
+   * Details about the Application that would use the Operator's Network APIs.
+   */
+  @TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+  @visibility("read", "create", "update")
+  configuredApplication: ApplicationProperties;
+
+  /**
+   * Details about the Organization owning the Application that would use the Operator's Network APIs.
+   */
+  @TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+  @visibility("read", "create", "update")
+  configuredApplicationOwner: ApplicationOwnerProperties;
+}
+
+/**
+ * Details about the Application Owner (i.e. organization) that would use the Operator's Network APIs.
+ */
+@TypeSpec.Versioning.added(Versions.v2024_11_01_preview)
+model ApplicationOwnerProperties {
+  /**
+   * Common name of the organization owning the application.
+   */
+  name: string;
+
+  /**
+   * Legal name of the organization owning the application.
+   */
+  legalName: string;
+
+  /**
+   * Trading name of the organization owning the application.
+   */
+  tradingName: string;
+
+  /**
+   * A description of the organization owning the application.
+   */
+  organizationDescription: string;
+
+  /**
+   * Unique Tax Number for the user's organization in the country/region the APC Gateway is being purchased.
+   */
+  taxNumber: string;
+
+  /**
+   * The type of organization that the Application Owner is.
+   */
+  organizationType: OrganizationType;
+
+  /**
+   * ID number for official identification of the organization.
+   */
+  organizationIdentificationId: string;
+
+  /**
+   * Issuing organization of the ID for official identification of the organization.
+   */
+  organizationIdentificationIssuer: string;
+
+  /**
+   * Type of official identification of the organization.
+   */
+  organizationIdentificationType: string;
+
+  /**
+   * Email address for the person responsible for the use of Network APIs in this application.
+   */
+  contactEmailAddress: string;
+
+  /**
+   * Legal representative of the organization.
+   */
+  legalRepresentative: Person;
```

Comment:
Apologies if I've missed some doc explaining this - we have privacyManager and legalRepresentative, are we intentionally leaving out dataProtectionOfficer?

Reply:
Only one of Privacy Manager and Data Protection Officer is mandatory - it's the user's choice which to provide for the Operate APIs. I've therefore simplified our API by just requiring one and not having the choice

Reply:
Ah yes, I remember discussing this in the original proposal doc now